apps/dashboard/lib/collections/index.ts

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { ratelimitNamespaces } from "./ratelimit_namespaces";
+import { ratelimitOverrides } from "./ratelimit_overrides";
+
+export const collection = {
+  ratelimitNamespaces,
+  ratelimitOverrides,
+};
+
+// resets all collections data and preloads new
+export function reset() {
```

Comment:
Make this async so we can await for better UX 

Then we can do a Promise.allSettled for preload.